src/main/java/apoc/nodes/Nodes.java

```diff
@@ -136,6 +139,80 @@ public boolean connected(@Name("start") Node start, @Name("start") Node end, @Na
         }
     }
 
+    @Procedure(mode = Mode.WRITE)
+    @Description("apoc.nodes.collapse([nodes...],[{properties:'overwrite' or 'discard' or 'combine'}]) yield from, rel, to merge nodes onto first in list")
+    public Stream<VirtualPathResult> collapse(@Name("nodes") List<Node> nodes, @Name(value = "config", defaultValue = "") Map<String, Object> config) {
+        if (nodes == null || nodes.isEmpty() || nodes.size() < 2) return Stream.empty();
```

Comment:
empty is included in < 2
and why empty stream with one node?
Shouldn't we then have a virtual node for one entry?